.github/workflows/pr-comment-terraspace.yaml

```diff
@@ -136,8 +136,8 @@ jobs:
         run: |
           res=$(curl "https://api.github.com/repos/${{ github.repository }}/pulls/${pr_number}/reviews?per_page=100" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -H 'Content-Type: application/json')
           echo "Res: $res"
-          echo "::set-output name=approval_state::$(echo "$res" | jq '.[] | select(.state=="APPROVED").state' | sed 's/"//g')"
-
+          # shellcheck disable=SC2086
+          echo "approval_state=$(echo $res | jq '.[] | select(.state=="APPROVED").state' | sed 's/"//g')" >> "$GITHUB_OUTPUT"
```

Comment:
There is a new context object that provides this we could make use of instead of making a REST call

Reply:
Done!